Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Magic doctrines

#### Changelog:
- Added magic doctrines that are needed for magic lifestyles from #121 
- Elunic faith also got special doctrine

#### How to test:
Make sure that doctrines:

1.) Assigned correctly and lore-friendly
- Idolised is the main magic type. On the other branches (NOT this one, this one is doctrines only), this doctrine will give bonuses to users of this magic, such as faster magic learning and opinion bonuses. Also it will be chosen more often by AI.

<details>
<summary>More</summary>

Faith can have only one idolised magic, and you need either to be in religious family of that magic type (e.g. idolising death magic is available for faith in death religious family) or with that magic syncretism tenet (e.g. death syncretism for idolising death magic).

![image](https://user-images.githubusercontent.com/46576860/107820897-a2da5a00-6d8c-11eb-89ae-93e2f442fcb0.png)

</details>

- Accepted is the type of magic that can used by characters of that faith.

<details>
<summary>More</summary>

You don't need anything to accept any form of magic

![image](https://user-images.githubusercontent.com/46576860/107821301-3e6bca80-6d8d-11eb-8685-0a80b2938f61.png)

</details>

- Ignored is the type of magic that is can't be used by characters of that faith, but users are not shunned or criminals.

<details>
<summary>More</summary>

You can't ignore magic that you can *possibly* idolise.

![image](https://user-images.githubusercontent.com/46576860/107821410-6f4bff80-6d8d-11eb-835a-22dec6c3422e.png)

</details>

- Shunned is the type of magic that is can't be used by characters of that faith, and shunned as in other criminal doctrines

<details>
<summary>More</summary>

You can't shun magic that you can *possibly* idolise.

![image](https://user-images.githubusercontent.com/46576860/107821643-de295880-6d8d-11eb-9563-9c26665452b6.png)

</details>

- Criminal is the type of magic that is can't be used by characters of that faith, and criminal as in other criminal doctrines

<details>
<summary>More</summary>

You can't criminalise magic that you can *possibly* idolise.

![image](https://user-images.githubusercontent.com/46576860/107821671-ee413800-6d8d-11eb-9fe5-ce9e39df465e.png)

</details>

2.) Work properly
- Tooltips should inform you *why* you can or can not pick this specific doctrine.
